geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -366,7 +373,33 @@ app.DrawController.prototype.onDrawEnd_ = function(event) {
         ol.geom.Polygon.fromCircle(featureGeom)
     );
   }
-  feature.set('name', 'element ' + (++this.featureSeq_));
+
+  /**
+   * @type {string}
+   */
+  var name;
+  switch (feature.getGeometry().getType()) {
+    case 'Point':
+      if (this.drawLabel.getActive()) {
+        name = this.translate_.getString('Label');
+      } else {
+        name = this.translate_.getString('Point');
+      }
+      break;
+    case 'LineString':
+      name = this.translate_.getString('LineString');
+      break;
+    case 'Polygon':
+      name = this.translate_.getString('Polygon');
+      break;
+  }
```

Comment:
The explicit listing of the feature types has more to do with allowing transifex to collect the strings for the different geom types .
with `this.translate_.getString(type)` gettext wont be able to collect all the possible geom type strings.